external/jacoco/test.properties

```diff
@@ -3,5 +3,5 @@ test_command="mvn clean verify"
 test_results="testResults"
 tag_version="v0.8.7"
 environment_variable="MODE=java"
-ubuntu_packages="git wget tar"
 maven_version="3.8.5"
```

Comment:
Can we move to 3.9.9?

Reply:
No way in this PR:( But otherwise I would love to.